.env.example

```diff
@@ -32,5 +32,7 @@ SPID_LOG_STORAGE_CONNECTION_STRING=DefaultEndpointsProtocol=http;AccountName=dev
 SPID_LOG_QUEUE_NAME=spidqueuename
 NOTIFICATIONS_STORAGE_CONNECTION_STRING=foobar
 NOTIFICATIONS_QUEUE_NAME=nqueuename
+USERS_LOGIN_STORAGE_CONNECTION_STRING=foobar
+USERS_LOGIN_QUEUE_NAME=ulqueuename
```

Comment:
Could you add these variables into the README

Reply:
Done